plugins/woocommerce/tests/php/src/Internal/Admin/Settings/PaymentsProviders/WooPayments/WooPaymentsServiceTest.php

```
@@ -8439,4 +8439,170 @@ private function get_woopayments_supported_countries(): array {
+		$stored_step_data = array(
+			'payment_methods' => array(
+				'apple_pay'  => 'false',
+				'google_pay' => 'false',
+				'card'       => 'true',
+			),
+		);
+
+		$this->mockable_proxy->register_static_mocks(
+			array(
+				Utils::class => array(
+					'get_nox_profile_onboarding_step_data_entry' => function ( $step_id, $location, $entry = null, $default_value = false ) use ( $stored_step_data ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$step_id'. (Unused Code Rules)

(UnusedFormalParameter)